src/lambda.js

```diff
@@ -9,8 +9,17 @@ const { promisify } = require('util');
 
 const LAMBDA_IMAGE = 'lambci/lambda:nodejs6.10';
 
-const createEnvironmentVariables = (environment) => Object.entries(environment)
-  .map(([ key, value ]) => `${key}=${value}`);
+const createEnvironmentVariables = (environment) => {
+  const envVariables = Object.entries(environment)
+    .map(([ key, value ]) => `${key}=${value}`);
+  if (environment.DISABLE_XRAY_LOGGING) {
+    // We want to disable xray logging in integration tests, because it's too verbose, see
+    // xray set up in lambda-runtime-tools
```

Comment:
`lambda-runtime-tools` is currently not part of our GitHub org so we might want to remove this reference.